### ISSUE TITLE
no way to tune the ClusterSingletonManager props underlying ClusterSharding

### ISSUE THREAD

**Issue by sclasen** [akka/akka#15330]
would be nice if you could tweak the `maxHandOverRetries` `maxTakeOverRetries`, `retryInterval` of the `ClusterSingletonManager` inside of `ClusterSharding`.

Mostly for testing purposes. Ran into this writing a multi-jvm test with cluster sharding and having to constantly wait out the default 10 seconds worth of ```
```
ClusterSingletonManager - Retry [10], sending HandOverToMe to [None]
```

Related...is there a good/test-kit-ish way to wait for ClusterSharding to be all the way available before doing stuff with it in tests?

**Comment by patriknw** [comment]
@sclasen Sorry that it took so long to get here, but now it is done. Will be released in 2.4-M2.

Everything can be configured per entity type, including the singleton settings for the coordinator. See https://github.com/akka/akka/commit/294659e2fe394782ca32fbffcc42953d932340eb if you are interested in details.

I also added `GetCurrentRegions` so that you can `awaitAssert` in the test until you have enough regions. See https://github.com/akka/akka/commit/c9a24478677a3d283b48478e37bea7d2a0e5363f

**Comment by sclasen** [comment]
:+1:
